api/views.py

```diff
@@ -323,7 +323,17 @@ class SecurityUserViewSet(ModelViewSet):
     queryset = SecurityUser.objects.all()
     permission_classes = [IsAuthenticated, IsAdminUser]
     authentication_classes = [FirebaseTokenAuthentication, ]
-    http_method_names = ['get', 'post']
+    http_method_names = ['get', 'post', 'put', 'delete']
+
+    def destroy(self, request, *args, **kwargs):
```

Comment:
is there a reason we need to override `destroy`?

Reply:
The default destroy method doesn't return a response, so I had to override it to return a message.